Peer dependency version issues when using with Astro 1.0.0-beta.x versions

npm ERR! Could not resolve dependency:
npm ERR! peer astro@">=0.23.0" from astro-imagetools@0.4.1
npm ERR! node_modules/astro-imagetools
npm ERR!   astro-imagetools@"^0.4.1" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: astro@0.26.1
npm ERR! node_modules/astro
npm ERR!   peer astro@">=0.23.0" from astro-imagetools@0.4.1
npm ERR!   node_modules/astro-imagetools
npm ERR!     astro-imagetools@"^0.4.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

--- a/packages/astro-imagetools/package.json
+++ b/packages/astro-imagetools/package.json
@@ -50,6 +50,6 @@
     "imagetools-core": "3.0.2"
   },
   "peerDependencies": {
-    "astro": ">=0.23.0"
+    "astro": ">=0.23.0 || >=1.0.0-beta.7"
   }
 }

